### PR TITLE
MGMT-3145 Reset CVO timeout in case of a change

### PR DESCRIPTION
**Pull request by YuviGold**

### CODE REVIEW

File: src/assisted_installer_controller/assisted_installer_controller.go [hunk 1]
```diff
@@ -556,9 +556,14 @@ func (c controller) validateConsolePod() bool {
...
 
 func (c controller) waitingForClusterVersion() error {
-	isClusterVersionAvailable := func() bool {
+	isClusterVersionAvailable := func(timer *time.Timer) bool {
 		newCvoStatus := c.validateClusterVersion()
 		if c.cvoStatus.isAvailable != newCvoStatus.isAvailable || (c.cvoStatus.message != newCvoStatus.message && newCvoStatus.message != "") {
+			if !timer.Stop() {
+				<-timer.C
+			}
+			timer.Reset(WaitTimeout)
```
Comment (romfreiman) [inline src/assisted_installer_controller/assisted_installer_controller.go:565]: what change are u looking at?

Comment (YuviGold) [reply src/assisted_installer_controller/assisted_installer_controller.go:565]: Message/Availability change.
Basically, that timeout indicates static-waiting and not on-going changes.

Comment (romfreiman) [reply src/assisted_installer_controller/assisted_installer_controller.go:565]: But message can be:
"Multiple Errors"....

Comment (YuviGold) [reply src/assisted_installer_controller/assisted_installer_controller.go:565]: We can start with this check and in the future make a smarter check. Unless you think it's necessary.

Comment (nmagnezi) [reply src/assisted_installer_controller/assisted_installer_controller.go:565]: @YuviGold do you have an assisted-service log with this running, for reference?

**Comment by YuviGold** [comment]
Timeout will be handled by the service.

**Pull request closed without merging**